features/volgindicaties.feature

```diff
@@ -0,0 +1,159 @@
+      En gebruiker "B" heeft een volgindicatie voor de persoon met burgerservicenummer "999990457" en einddatum "2021-06-30"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999990457" en geen einddatum
+      Dan stuurt de update applicatie GEEN Ap01 bericht met burgerservicenummer "999990457" naar het GBA-V
+
+   Scenario: Een gebruiker zet een volgindicatie op een persoon die niet meer gevolgd wordt door een andere gebruiker
+      Gegeven de persoon met burgerservicenummer "999992855" is NIET ingeschreven in deze gemeente
+      En gebruiker "B" heeft een volgindicatie voor de persoon met burgerservicenummer "999992855" en einddatum "2021-03-01"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999992855" en geen einddatum
+      Dan stuurt de update applicatie een Ap01 bericht met burgerservicenummer "999992855" naar het GBA-V
+    
+   Scenario: Een gebruiker verlengt de einddatum van de volgindicatie
+      Gegeven de persoon met burgerservicenummer "999992806" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999992806" en einddatum "2021-03-15"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999992806" met einddatum "2021-06-30"
+      Dan stuurt de update applicatie GEEN Ap01 bericht met burgerservicenummer "999992806" naar het GBA-V
+
+   Scenario: Een gebruiker wijzigt de einddatum van de volgindicatie naar vandaag
+      Gegeven de persoon met burgerservicenummer "999994220" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999994220" en einddatum "2021-03-15"
+      En geen enkele andere gebruiker heeft een volgindicatie voor de persoon met burgerservicenummer "999994220"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999994220" met einddatum "2021-03-05"
+      Dan stuurt de update applicatie een Av01 bericht met burgerservicenummer "999994220" naar het GBA-V
+
+   Scenario: Een gebruiker wijzigt de einddatum van de volgindicatie naar een datum in het verleden
+      Gegeven de persoon met burgerservicenummer "999994220" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999994220" en geen einddatum
+      En geen enkele andere gebruiker heeft een volgindicatie voor de persoon met burgerservicenummer "999994220"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999994220" met einddatum "2021-03-05"
+      Dan stuurt de update applicatie een Av01 bericht met burgerservicenummer "999994220" naar het GBA-V
+
+   Scenario: Een gebruiker beëindigt de volgindicatie die ook voor alle andere gebruikers inactief is
```

Comment:
Wat is het verschil tussen beëindigen van een volgindicatie en het zetten van de einddatum naar een datum in het verleden? Volgens mij is de actie hetzelfde. Ik zou dus één van de twee kiezen en die dan voor alle scenario's gebruiken waar een volgindicatie wordt beëindigd